### DISCUSSION TITLE
meditation/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/shane0.github.io#36] [General]
# meditation/

https://shane0.github.io/meditation/

**Comment by shane0** [comment]
# slides

- add a slide for some daily meditations
- meditative mind
- anapanasati

**Reply by shane0** [reply]
- done: https://shane0.github.io/daily.html#/title-slide
- could add a pipeline for this later
- to update run `./utils/daily.sh`

```sh
#!/usr/bin/env bash
pandoc -t revealjs -s -o ./slides/daily.html ./slides/daily.md -V revealjs-url=https://unpkg.com/reveal.js@3.9.2/ -V theme=solarized
cp ./slides/daily.html ./docs/
```

**Comment by shane0** [comment]
# meditation experiments

https://github.com/shane0/shane0.github.io/discussions/37

**Reply by shane0** [reply]
| Unique Vajrayana Meditations     |
|---------------------------------|
| Sky Gazing                      |
| Fire Visualization              |
| Deity Yoga                      |
| Tummo (Inner Heat) Meditation   |
| Dream Yoga                      |
| Sound Bath Meditation           |
| Mandala Meditation              |
| Guru Yoga                       |
| Chod                            |
| Chanting Mantras                |
| Subtle Energy Meditation        |

**Reply by shane0** [reply]
| Unique Mahayana Meditations     |
|--------------------------------|
| Loving-Kindness Meditation     |
| Tonglen Meditation             |
| Buddha Nature Contemplation     |
| Emptiness Meditation           |
| Metta (Compassion) Meditation  |
| Repentance Meditation          |
| Mantra Recitation              |
| Pure Land Visualization        |
| Visualization of Bodhisattvas  |
| Prajnaparamita Meditation      |
| Avalokiteshvara Meditation     |
| Diamond Sutra Contemplation    |

**Reply by shane0** [reply]
| Unique Theravada Meditations      |
|----------------------------------|
| Vipassana Meditation              |
| Anapanasati (Mindfulness of Breath) Meditation |
| Metta (Loving-Kindness) Meditation |
| Kasina Meditation                 |
| Walking Meditation                |
| Body Scan Meditation              |
| Four Elements Meditation          |
| Death Contemplation               |
| Loving-Kindness towards Difficult People |
| Reflection on Impermanence        |
| Mindfulness of Daily Activities    |

**Reply by shane0** [reply]
| Unique Zen Meditations           |
|---------------------------------|
| Zazen (Seated Meditation)       |
| Koan Practice                   |
| Shikantaza (Just Sitting)       |
| Walking Meditation              |
| Chanting Meditation             |
| Tea Ceremony Meditation         |
| Samu (Work Meditation)          |
| Kinhin (Walking Meditation)     |
| Hua Tou (Word Head) Meditation  |
| Noh Theatre Meditation          |

**Reply by shane0** [reply]
| Unique Theravada Meditations     |
|---------------------------------|
| Vipassana Meditation             |
| Anapanasati (Mindfulness of Breath) Meditation |
| Metta (Loving-Kindness) Meditation |
| Kasina Meditation                |
| Walking Meditation               |
| Body Scan Meditation             |
| Contemplation of the Body        |
| Contemplation of Feelings       |
| Contemplation of Mind           |
| Contemplation of Mental Objects |
| Loving-Kindness towards All Beings |
| Reflection on Impermanence       |
| Reflection on the Four Noble Truths |

**Reply by shane0** [reply]
| Unique Yogacara Meditations |
|----------------------------|
| Mindfulness of Cognition   |
| Contemplation of Mind-Only |
| Perception of Interdependence |
| Meditating on the Luminous Nature of Consciousness |
| Visualization of Pure Lands |
| Contemplation on the Eight Consciousnesses |

**Reply by shane0** [reply]
started linji

**Comment by shane0** [comment]
# yapping with chatgpt

- dispel the illusion of an enduring self or ego
- dependent origination or dependent arising
- everything we encounter in our lives is the result of a complex web of causes and conditions coming together
- all conditioned phenomena are subject to impermanence (anicca)
- impermanence is a fundamental characteristic of conditioned existence
-  all phenomena arise and exist due to causes and conditions
- dismantle the illusion of a fixed, independent self and deepens our understanding of impermanence and interdependence

## conditioned

| Conditioned Phenomena |
|----------------------|
| Thoughts             |
| Emotions             |
| Sensations           |
| Physical Form        |
| Perceptions          |
| Consciousness        |
| Volitions            |
| Mental Formations    |

## unconditioned

| Unconditioned Phenomena |
|------------------------|
| Nirvana (Nibbana)      |
| Deathless (Amata)      |
| Unborn (Ajāta)         |
| Uncreated (Akata)      |

---

| Dependent Origination                                                |
|----------------------------------------------------------------------|
| Ignorance leads to Mental Formations (Sankhara)                       |
| Mental Formations lead to Consciousness (Vijnana)                     |
| Consciousness leads to Mind and Body (Nama-Rupa)                      |
| Mind and Body lead to Six Sense Bases (Salayatana)                    |
| Six Sense Bases lead to Contact (Phassa)                               |
| Contact leads to Feeling (Vedana)                                     |
| Feeling leads to Craving (Tanha)                                      |
| Craving leads to Clinging (Upadana)                                   |
| Clinging leads to Becoming (Bhava)                                    |
| Becoming leads to Birth (Jati)                                        |
| Birth leads to Aging and Death (Jara-Marana)                          |
| Aging and Death lead to Suffering (Dukkha)                            |
| Suffering leads to Clinging (Upadana)                                  |
| Clinging leads to Becoming (Bhava)                                    |
| Becoming leads to Birth (Jati)                                        |
| Birth leads to Ignorance and the cycle continues (Samsara)             |

---

| Skillful Means                                                     |
|-------------------------------------------------------------------|
| Generosity                                                        |
| Moral Discipline                                                  |
| Patience                                                          |
| Diligence                                                         |
| Meditative Concentration                                          |
| Wisdom                                                            |
| Compassion                                                        |
| Loving-Kindness                                                   |
| Skillful Speech                                                    |
| Skillful Action                                                    |
| Skillful Livelihood                                               |
| Skillful Effort                                                   |
| Skillful Mindfulness                                              |
| Skillful Understanding                                             |
| Skillful Thought                                                   |
| Skillful Intention                                                 |
| Skillful View                                                      |
| Skillful Reflection                                                |
| Skillful Application of the Noble Eightfold Path                    |
| Skillful Application of the Four Noble Truths                      |
| Skillful Means for the Benefit and Liberation of Beings             |
| Skillful Means for Cultivating Virtuous Qualities and Attainments  |
| Skillful Means for Reducing Suffering and Promoting Happiness      |
| Skillful Means for Cultivating Wisdom and Compassion               |
| Skillful Means for Nurturing Spiritual Growth and Liberation       |
| Skillful Means for Cultivating Mindfulness and Awareness           |
| Skillful Means for Developing Insight and Enlightenment            |

---

| Meditative Mind                 | Afflicted Mind               |
|--------------------------------|------------------------------|
| Concentration                  | Distraction                  |
| Samatha (Tranquility)          | Restlessness                 |
| Vipassana (Insight)            | Confusion                    |
| Single-Pointedness             | Attachment                   |
| Equanimity                     | Aversion                     |
| Calmness                       | Anger                        |
| Stillness                      | Greed                        |
| Clarity                        | Ignorance                    |
| Alertness                      | Delusion                     |
| Mindfulness                    | Anxiety                      |
| Awareness                      | Fear                         |
| Non-attachment                 | Jealousy                     |
| Non-judgment                   | Resentment                   |
| Non-reactivity                 | Hatred                       |
| Inner Silence                  | Envy                         |
| Serenity                       | Pride                        |
| Presence                       | Restlessness                 |
| Deep Relaxation                | Impatience                   |
| Absorption                     | Worry                        |
| Mental Stability               | Doubt                        |
| Insightful Observation         | Guilt                        |
| Contemplative Inquiry          | Regret                       |
| Liberation from Distractions   | Agitation                    |
| Focused Attention              | Insecurity                   |
| Balanced Mental States         | Inadequacy                   |
| Cultivation of Positive Qualities                           |
| Deepening of Wisdom and Understanding                       |
| Nurturing of Inner Peace and Happiness                      |
| Expansion of Consciousness and Insight                      |
| Cultivation of Inner Well-being and Spiritual Growth        |
| Enhancement of Self-Realization and Self-Transformation   |

**Comment by shane0** [comment]
# varieties

Country | Type
-- | --
Sri Lanka | Theravada Buddhism
Myanmar | Theravada Buddhism
Thailand | Theravada Buddhism
Cambodia | Theravada Buddhism
Laos | Theravada Buddhism
Vietnam | Mahayana Buddhism
China | Mahayana Buddhism
Korea | Mahayana Buddhism
Japan | Mahayana Buddhism
Mongolia | Vajrayana Buddhism
Tibet | Vajrayana Buddhism
Bhutan | Vajrayana Buddhism
Nepal | Vajrayana Buddhism

**Comment by shane0** [comment]
# aimlessness

- a rose is fine the way it is, does not have to strive to be better, same applies to a human
- the way you were born is enough achievement
- stop: measuring days by degree of productivity
- start: experiencing days by degree of presence

**Comment by shane0** [comment]
# koans

- dwelling nowhere mind comes forth
- form is emptiness and emptiness is form

## mirror

A monk asked a Zen master, "What is the meaning of no mirror?"

The Zen master replied, "No mirror, no reflection."

The monk was confused. "But how can there be no reflection if there is no mirror?" he asked.

The Zen master smiled. "The mirror is the mind," he said. "The reflection is the world. When the mind is clear, there is no need for a mirror. The world is reflected perfectly in the mind."

## location

A monk was walking through the forest when he came across a Zen master sitting under a tree. The monk asked the master, "Where is the best place to dwell?"

The master replied, "Dwell nowhere."

The monk was confused. "But if I don't dwell anywhere, where will I live?" he asked.

The master smiled. "The world is your home," he said. "There is no need to dwell anywhere else."]

## emptiness form

zen reader: 

“MATTER AND EMPTINESS - SEEING MATTER itself as emptiness produces great wisdom so one does not dwell in birth and death; seeing emptiness as equivalent to matter produces great compassion so one does not dwell in nirvana. Yun-feng”

### nlp

The koan "Emptiness is form" is a Zen teaching that encourages us to see the world as it really is, without any preconceptions or attachments. It is a reminder that all things are interconnected and that there is no separation between the observer and the observed. When we can see this, we can experience the world with a sense of wonder and awe. We can also let go of our attachments and live in the present moment.

The koan "Emptiness is form, form is emptiness" is a Zen teaching that encourages practitioners to see the world as it really is, without any preconceptions or attachments. It is a reminder that all things are empty of inherent existence, and that our experience of the world is created by our own minds.

The koan can be interpreted in many ways. Some people see it as a call to live a life of detachment and non-duality, while others see it as a call to find peace and contentment in the present moment. Ultimately, the meaning of the koan is up to the individual to decide.

Here is a story that illustrates the koan of emptiness is form:

> A monk was walking through the forest when he came across a Zen master sitting under a tree. The monk asked the master, "What is the meaning of emptiness is form?"

>

> The master replied, "Form is emptiness, and emptiness is form."

>

> The monk was confused. "But how can form be emptiness, and emptiness be form?" he asked.

>

> The master smiled. "Everything in the world is made up of atoms," he said. "These atoms are constantly moving and changing. When we look at something, we see a form, but this form is just an illusion. The true nature of things is emptiness."

The monk was still confused, but he decided to meditate on the master's words. After a while, he began to understand what the master meant. He realized that all things are impermanent and constantly changing. He also realized that the mind is the source of all experience. When we can see the world as it really is, without any preconceptions or attachments, we can experience the freedom and peace of emptiness.

The koan of emptiness is form is a reminder that everything in the universe is interconnected. There is no separation between the observer and the observed. When we can see this, we can experience the world with a sense of wonder and awe. We can also let go of our attachments and live in the present moment.

**Comment by shane0** [comment]
# science & buddhism

Sure. Here is a summary of Buddhism and its 20 most similar laws in science:

* **The Three Universal Truths:** The Three Universal Truths are the law of impermanence, the law of suffering, and the law of non-self.
    * The law of impermanence states that everything in the universe is constantly changing.
    * The law of suffering states that all life is characterized by suffering.
    * The law of non-self states that there is no separate, unchanging self.
* **The Four Noble Truths:** The Four Noble Truths are the truth of suffering, the truth of the cause of suffering, the truth of the end of suffering, and the truth of the path to the end of suffering.
    * The truth of suffering states that all life is characterized by suffering.
    * The truth of the cause of suffering states that the cause of suffering is craving.
    * The truth of the end of suffering states that the end of suffering is possible through the cessation of craving.
    * The truth of the path to the end of suffering states that the path to the end of suffering is the Eightfold Path.
* **The Eightfold Path:** The Eightfold Path is a set of ethical and spiritual guidelines that can help people to end suffering and achieve enlightenment.
    * Right understanding
    * Right thought
    * Right speech
    * Right action
    * Right livelihood
    * Right effort
    * Right mindfulness
    * Right concentration

The laws of science that are most similar to Buddhism are those that deal with the nature of reality, the cause of suffering, and the path to liberation. These laws include:

* The law of conservation of energy
* The law of cause and effect
* The law of entropy
* The law of evolution
* The law of karma

These laws suggest that the universe is a vast and interconnected system, that our actions have consequences, that change is inevitable, that life is a process of evolution, and that our thoughts and actions create our own reality.

Buddhism and science offer complementary perspectives on the nature of reality. Buddhism provides a spiritual and ethical framework for understanding the laws of nature, while science provides a physical and material explanation for the laws of nature. Together, they can help us to live more fulfilling and meaningful lives.

**Comment by shane0** [comment]
# amata

 "Of all the footprints, that of the elephant is supreme. Similarly, of all mindfulness meditation, that on death is supreme." (AN 6.19)

https://www.dhammatalks.org/suttas/AN/AN6_19.html

——

mentions effluents?

In Buddhism, the term "effluents" (Pali: asava) refers to the three root causes of suffering:

* **Sensuality:** This refers to our attachment to sense pleasures, such as food, sex, and material possessions.
* **Becoming:** This refers to our craving for existence, whether in this world or in a future world.
* **Ignorance:** This refers to our misunderstanding of the true nature of reality, which leads to fear, anxiety, and other negative emotions.

The Buddha taught that the end of suffering is possible by ending the effluents. This can be done through the practice of the Eightfold Path, which is a set of guidelines for living a moral, ethical, and mindful life.

The Buddha's teaching on the effluents is based on the principle of dependent origination (Pali: paticca samuppada), which states that all phenomena are interconnected and interdependent. This means that our suffering is not caused by any one thing, but by a complex web of causes and conditions.

By understanding the nature of the effluents and the principle of dependent origination, we can begin to see how our suffering is created. This knowledge can then be used to develop the wisdom and insight necessary to end the effluents and achieve liberation from suffering.

Here are some specific practices that can help us to end the effluents:

* **Metta meditation:** This is a meditation practice that cultivates loving-kindness and compassion. Metta meditation can help us to let go of our attachment to self and to see others as our equals.
* **Vipassana meditation:** This is a meditation practice that cultivates insight into the true nature of reality. Vipassana meditation can help us to see the impermanence, unsatisfactoriness, and non-self nature of all phenomena.
* **The Eightfold Path:** This is a set of guidelines for living a moral, ethical, and mindful life. The Eightfold Path can help us to develop the wisdom and insight necessary to end the effluents.

Ending the effluents is not an easy task, but it is possible with sustained practice. The Buddha's teachings on the effluents offer a clear path to liberation from suffering. If you are interested in learning more about this topic, I recommend reading the following texts:

* **The Heart of the Buddha's Teaching** by Thich Nhat Hanh
* **The Path to Liberation** by Bhikkhu Bodhi
* **Dependent Origination** by Thanissaro Bhikkhu

**Comment by shane0** [comment]
- [react js page with cards](https://nucamp-demos.vercel.app/meditations)

**Comment by shane0** [comment]
https://gist.github.com/shane0/a9a62b532f0bbd0e091ceaa63cb73b0e

**Comment by shane0** [comment]
- https://shanenull.com/buddhism/samu/
- https://shanenull.com/buddhism/lesson_1/
- https://shanenull.com/buddhism/record/
- https://shanenull.com/buddhism/linji/
- https://shanenull.com/buddhism/heart_sutra/
- https://shanenull.com/buddhism/anapanasati/
